src/darwin/Framework/CHIP/MTRDeviceControllerXPCConnection.h

```diff
@@ -41,16 +45,15 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * This method is just for test purpsoe.
  */
-+ (instancetype)connectionWithWorkQueue:(dispatch_queue_t)workQueue connectBlock:(NSXPCConnection * (^)(void) )connectBlock;
++ (instancetype)connectionWithWorkQueue:(dispatch_queue_t)workQueue connectBlock:(MTRXPCConnectBlock)connectBlock;
 
-- (void)getProxyHandleWithCompletion:(void (^)(dispatch_queue_t queue,
-                                         MTRDeviceControllerXPCProxyHandle * _Nullable container))completion;
+- (void)getProxyHandleWithCompletion:(MTRGetProxyHandleHandler)completion;
 - (void)registerReportHandlerWithController:(id<NSCopying>)controller
-                                     nodeId:(NSUInteger)nodeId
-                                    handler:(void (^)(id _Nullable values, NSError * _Nullable error))handler;
+                                     nodeId:(NSNumber *)nodeId
```

Comment:
`nodeID:` ?

Reply:
That's changing in an upcoming PR that changes all the "Id" bits to "ID" across the framework, where possible.